lib/libutils/ext/arch/arm/mcount_a32.S

```diff
@@ -26,7 +26,7 @@
  */
 FUNC __gnu_mcount_nc, :
 	stmdb		sp!, {r0-r3, lr}
-#ifdef CFG_TA_GPROF_SUPPORT
+#if defined(CFG_TA_GPROF_SUPPORT) && !defined(__KERNEL__)
```

Comment:
At this point (I mean at commit "ftrace: move ftrace code from libutee to libutils"), the mcount functions are not needed at all for kernel code, so I think this line does not need to be changed and the file should be excluded from kernel build instead (see below in `sub.mk`).

Reply:
Okay, will split current patch as follows:
1. `ftrace: move ftrace code from libutee to libutils`
2. `ftrace: core: prepare support for syscall ftrace`